Modal buy now vadim

VadBorovec opened the pull request:
Додає модалку на Buy Now
Скрипт на Disabled button
В секції Hero-Header робив прив'язку до своєї модалки.
В форму добавив різні типи валідації та псевдокласи стану.

VadBorovec committed:
add modal buy now

without position and js

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
Add disabled-btn.js and modal buy now

VadBorovec commented:
щойно побачив що Микола також робив пулл на секцію Hero-Header. Якщо будуть конфлікти, при потребі зможу зробити без змін в секції Hero-Header. Там буквально одні фразу змінив на кнопці:
` js-open-modal" data-modal="modal-buy-now"` замість `data-menu-open`

VadBorovec commented:
скрипт добавив в кінці своєї секції...потрібно буде перенести на index.html

KostiantynU reviewed (changes requested):
1. В модалці Н2 - Н2 тільки для секцій, давай тег Р
2. Поправити картинки - одна і та ж картинка вказана для Х2
3. Сама картинка має розмір 128х132. Хоча у тега імж картинки вкладені всі вірного розміру. Я знайшов причину такоїх поведінки - бордер для Р забирає 2рх зліва, 2рх зправа, і картинка стає 132рх, як по макету. Може той бордер дати, лішці?
4. Кнопка бачу неактивна, але якщо вибрати морозиво - так і залишається. Схоже ще не все підключено? У твоїй гілці працює? Бо я в ній і дивлюсь...

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
remove large padding on section contacts

also
at modal Buy Now:
1.edit h2 to p
2. correct x1, x2 images

at hero-header - correct classes on buttons, for open modal Buy Now

VadBorovec commented:
1. В модалці Н2 - Н2 тільки для секцій, давай тег Р
замінив

2. Поправити картинки - одна і та ж картинка вказана для Х2
виправив

3. Сама картинка має розмір 128х132. Хоча у тега імж картинки вкладені всі вірного розміру. Я знайшов причину такоїх поведінки - бордер для Р забирає 2рх зліва, 2рх зправа, і картинка стає 132рх, як по макету. Може той бордер дати, лішці?
не вийшло, може завтра на свіжу голову спробую

4. Кнопка бачу неактивна, але якщо вибрати морозиво - так і залишається. Схоже ще не все підключено? У твоїй гілці працює? Бо я в ній і дивлюсь...
Кнопка активується на заповнення інпуту - Ім'я

Пробіл між секціями Контакти / Футер прибрав

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
add anchors

VadBorovec committed:
move js to index

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
transform: translate(-50%, -50%)

 transform: translate(-50%, -50%)
from modal

VadBorovec reviewed (commented):
внесені правки

VadBorovec committed:
Merge branch 'main' into modal-buy-now-Vadim

VadBorovec committed:
Добавляє верхній паддінг на Customer review

VadBorovec committed:
Final commit

VadBorovec committed:
Final final
